open_spiel/integration_tests/api_test.py

```diff
@@ -284,6 +284,29 @@ def setUpClass(cls):
             depth_limit=7).values())
     cls.game_type = cls.game.get_type()
 
+  def test_sequence_lengths(self):
+    max_history_len = self.game.max_history_length()
+    max_move_number = self.game.max_move_number()
+    max_chance_nodes_in_history = self.game.max_chance_nodes_in_history()
```

Comment:
Can you put this into a try / except block, and just return if max_chance_nodes_in_history fails?